Project.toml

```
@@ -1,6 +1,6 @@
 name = "ChainRulesTestUtils"
 uuid = "cdddcdb0-9152-4a09-a978-84456f9df70a"
-version = "0.6.0-DEV"
+version = "0.6.1"
```

Can't make a patch release til we do an actual release
but lets do an actual release with this
```suggestion
version = "0.6.0"
```

---

Actually can keave it at 0.6.1 and I am releasing 0.6 presently